packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -407,11 +408,23 @@ export default class ImageEdit implements EditorPlugin {
         return element;
     }
 
-    private insertImageWrapper(image: HTMLImageElement, wrapper: HTMLSpanElement, scale: number) {
+    private insertImageWrapper(
+        editor: IEditor,
+        image: HTMLImageElement,
+        wrapper: HTMLSpanElement,
+        scale: number
+    ) {
         this.zoomWrapper = this.copyImageSize(image, this.createZoomWrapper(wrapper, scale));
+        const editorDiv = this.getEditorDiv(editor);
+        this.zoomWrapper.style.zIndex = `${getBiggestZIndex(editorDiv) + 1}`;
         this.editor.getDocument().body.appendChild(this.zoomWrapper);
     }
 
+    private getEditorDiv(editor: IEditor) {
```

Comment:
Let's try to avoid getting editor div directly. Plugin should not know editor DIV at all.

You can use editor.getScrollContainer() as the first element.